templates/gRPC-Core.podspec.template

```diff
@@ -192,12 +192,12 @@
       ss.header_mappings_dir = '.'
       ss.libraries = 'z'
       ss.dependency "#{s.name}/Interface", version
-      ss.dependency 'BoringSSL-GRPC', '0.0.7'
+      ss.dependency 'BoringSSL-GRPC', '0.0.8'
       abseil_version = '1.20200225.0'
       % for abseil_spec in grpc_abseil_specs:
       ss.dependency '${abseil_spec}', abseil_version
       % endfor
-      ss.compiler_flags = '-DGRPC_SHADOW_BORINGSSL_SYMBOLS'
+      ss.compiler_flags = '-DBORINGSSL_PREFIX=GRPC'
```

Comment:
qq: have you checked that the symbols are actually getting prefixed when you build like this? I don't see the boringssl_prefix_symbols.h being used anywhere in our build files so I'm just wondering what the trick is. Perhaps boringssl_prefix_symbols.h is being included by boringssl headers, which makes it transparent for grpc usage of boringssl?

Reply:
Yes I did check the symbols are prefixed, and correctly used by gRPC. The `boringssl_prefix_symbols.h` is used [in BoringSSL headers](https://github.com/google/boringssl/blob/538a124d7073eba59e8841a0b88bc38aa1fe95aa/include/openssl/base.h#L78-L80). We just need to define the `BORINGSSL_PREFIX` macro when building gRPC-Core and BoringSSL-GRPC.